app.go

```diff
@@ -12,6 +12,8 @@ type app struct {
 	chapter int
 
 	err error
+
+	exitSignal chan bool
```

Comment:
Do you think it would be simpler to just use a boolean rather than a channel of booleans? Currently (other than the termbox event loop) goreader is single-threaded, so I'm not sure we gain much by using a channel.

Reply:
I had the same debate in my mind, and this somehow violates YAGNI principle.
That been said, I thought it wasn't big deal to have a channel so we could exit the app from other threads.
Let's say in the future we could have a bookmarks manager or some other feature, so this could be a way to send the exit signal.
But if you prefer we could use a boolean for now.